k8s/upgrade/src/bin/upgrade-job/upgrade/utils.rs

```diff
@@ -74,6 +73,9 @@ pub(crate) async fn rebuild_result(
                 }
             }
         }
+        if volume_over_nodes.is_empty() {
+            break 'outermost;
```

Comment:
why do we need to specify it, aren't we inside only outermost here?

Reply:
Just for readability. I can remove it.

Reply:
Removed loop label and break targets.

Reply:
It's fine to keep, just got confused why it was there.